EIPS/eip-7516.md

```diff
@@ -0,0 +1,67 @@
+---
+eip: 7516
+title: BLOBBASEFEE opcode
+description: Opcode that returns the current data-blob base-fee
+author: Carl Beekhuizen (@carlbeek)
+discussions-to: https://ethereum-magicians.org/t/eip-75xx-blobbasefee/15761
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-09-11
+requires: 3198, 4844
+---
+
+## Abstract
+
+Add a `BLOBBASEFEE (0x49)` that returns the value of the blob base-fee of the current block it is executing in. It is the identical to [EIP-3198](./eip-3198.md) (`BASEFEE` opcode) except that it returns the blob base-fee as per [EIP-4844](./eip-4844.md).
+
+## Motivation
+
+The intended use case would be for contracts to get the value of the blob base-fee. This feature enables blob-data users to programmatically account for the blob gas price, eg:
+
+- Allow rollup contracts to trustlessly account for blob data usage costs.
+- Blob gas futures can be implemented based on it which allows for blob users to smooth out data blob costs.
+
+## Specification
+
+Add a `BLOBBASEFEE` opcode at `(0x49)`, with gas cost `G_base`.
```

Comment:
WTF is `G_base`? EIP-4844 does not define `G_base` nor "blob base-fee".

Reply:
clarified in #7701:

`G_base` was a hold over from starting with EIP-3198 and is not relevant to this EIP.

"blob base-fee" is the colloquial name given to the result of the function `get_blob_gasprice(header)` as defined in EIP-4844.